### DISCUSSION TITLE
Check if mesh is correctly done

### DISCUSSION THREAD

**Discussion by francescomosconi** [SeisSol/SeisSol#784] [Q&A] [answered]
Hi, 

do you have any suggestion on how to check if the mesh file is correctly done?

I'm currently using gmsh.

Thanks!

**Comment by Thomas-Ulrich** [comment, marked as answer]
if you compile the latest seissol, it will raise an error if the boundaries are not correctly set.

**Reply by Thomas-Ulrich** [reply]
if you want to do it outside seissol use https://github.com/SeisSol/Meshing/tree/master/check_mesh

**Reply by francescomosconi** [reply]
Thanks for the suggestion!
